pages/index.jsx

```diff
@@ -23,6 +25,9 @@ var Index = React.createClass({
           <div id="my-app" dangerouslySetInnerHTML={{__html: this.props.markup}}></div>
           <script src="https://cdn.polyfill.io/v1/polyfill.min.js?features=Intl.~locale.fr,Intl.~locale.pt-BR,Intl.~locale.de,Intl.~locale.id,Intl.~locale.es,Intl.~locale.en-US"></script>
           <script src="/main.js"></script>
+          <Pontoon/>
+          <OptimizelySubdomain/>
+          <Optimizely/>
```

Comment:
I have double checked with @adamlofting and optimizely needs to be in the head, and needs to load sync.

Otherwise we see a flash of the old test.